basic/src/test/java/org/jboss/hal/testsuite/test/configuration/datasources/DomainTestConnectionTestCase.java

```diff
@@ -1,112 +1,82 @@
 import org.junit.AfterClass;
-import org.junit.Before;
-import org.junit.BeforeClass;
-import org.junit.Ignore;
 import org.junit.Test;
 import org.junit.experimental.categories.Category;
 import org.junit.runner.RunWith;
-import org.wildfly.extras.creaper.core.CommandFailedException;
 import org.wildfly.extras.creaper.core.online.OnlineManagementClient;
+import org.wildfly.extras.creaper.core.online.operations.Address;
+import org.wildfly.extras.creaper.core.online.operations.Operations;
 import org.wildfly.extras.creaper.core.online.operations.admin.Administration;
 
 import java.io.IOException;
 import java.util.concurrent.TimeoutException;
 
-/**
- * @author jcechace
- */
-@Ignore("Test connection button has been removed from datasource wizard - JBEAP-2821")
 @Category(Domain.class)
 @RunWith(Arquillian.class)
 public class DomainTestConnectionTestCase extends AbstractTestConnectionTestCase {
 
     @Page
-    private DomainConfigurationPage domainConfigurationPage;
+    private DatasourcesPage datasourcesPage;
 
-    private static String dsNameValid;
-    private static String dsSameNameValid;
-    private static String dsSameNameInvalid;
-
-
-    private static final String VALID_URL = "jdbc:h2:mem:test2;DB_CLOSE_DELAY=-1";
-    private static final String INVALID_URL = "invalidUrl";
+    private static final String
+            VALID_URL = "jdbc:h2:mem:test2;DB_CLOSE_DELAY=-1",
+            INVALID_URL = "invalidUrl";
 
     private static OnlineManagementClient client = ManagementClientProvider.createOnlineManagementClient();
     private static OnlineManagementClient fullHaClient = ManagementClientProvider.withProfile("full-ha");
     private static Administration administration = new Administration(client);
+    private static Operations fullHaOperations = new Operations(fullHaClient);
     private static DataSourcesOperations dsOps = new DataSourcesOperations(client);
-    private static DomainManager manager = new DomainManager(CliClientFactory.getClient());
-
-    // Setup
-    @BeforeClass
-    public static void setup() throws CommandFailedException, InterruptedException, TimeoutException, IOException {  // create needed datasources
-        dsNameValid = dsOps.createDataSource(VALID_URL);
-        dsSameNameValid = new DataSourcesOperations(fullHaClient, "full-ha").createDataSource(VALID_URL); ///
-        dsSameNameInvalid = dsOps.createDataSource(dsSameNameValid, INVALID_URL);
-        administration.reload();
-    }
 
     @AfterClass
-    public static void tearDown() throws CommandFailedException { // remove datasources when finished
+    public static void tearDown() throws InterruptedException, TimeoutException, IOException {
         try {
-            dsOps.removeDataSource(dsNameValid);
-            new DataSourcesOperations(fullHaClient, "full-ha").removeDataSource(dsSameNameValid); ///
-            dsOps.removeDataSource(dsSameNameInvalid);
+            administration.reloadIfRequired();
```

Comment:
??? teardown method which calls only reload seems as hack for something which should be already handled, but OK.